netlib/tcp.py

```diff
@@ -369,14 +372,14 @@ def finish(self):
     """
 
     def _create_ssl_context(self,
-                            method=SSLv23_METHOD,
+                            method=TLSv1_2_METHOD,
```

Comment:
That shouldn't be required?
`SSLv23_METHOD` + `OP_NO_SSLv2 | OP_NO_SSLv3` == TLS 1.0+, which is what we want to keep for now.